sharkfuser/include/fusilli/attributes/tensor_attributes.h

```diff
@@ -381,6 +382,24 @@ class TensorAttr {
     return expectedStride == stride_;
   }
 
+  // Convert logical dims + stride into physical dims
+  //  dims [N, C, H, W] + strideOrder [3, 2, 1, 0] -> [N, C, H, W]
+  //  dims [N, C, H, W] + strideOrder [3, 0, 2, 1] -> [N, H, W, C]
+  std::vector<int64_t> getPhysicalDim() const {
+    size_t numDims = dim_.size();
+    std::vector<int64_t> physicalDims(numDims);
+    std::vector<size_t> strideOrder(numDims);
```

Comment:
```suggestion
    std::vector<size_t> strideOrder;
```
as this is just assigned to the results of `get*StrideOrder`